src/TestProblem/Hydro/CCSN/Flag_CCSN.cpp

```diff
@@ -62,7 +62,10 @@ bool Flag_CoreCollapse( const int i, const int j, const int k, const int lv, con
    }
 
 
-// (2) always refined to highest level in the region with r < 30 km, if allowed
+// (2) always refined to highest level in the region with r < 30 km
+   if (  !MaxRefine  &&  ( r < amr->dh[lv] )  &&  ( amr->dh[lv] * sqrt(0.75) * UNIT_L >= 3e6 )  )
+      Flag = true;
+
    if (  !MaxRefine  &&  ( r * UNIT_L < 3e6 )  )
```

Comment:
I would prefer to merge the condition `!MaxRefine` in these two conditions, and separate the note. The purpose of this condition the one shown above, but the new condition above is to refine the innermost region recursively in order to trigger this one.

Reply:
I removed the previous condition and merge to one `if statement`